### ISSUE TITLE
Re-think Travis linting ruleset criteria 

### ISSUE THREAD

**Issue by gregrickaby** [WebDevStudios/wd_s#331]
This time, Travis is [bitching about](https://travis-ci.org/WebDevStudios/wd_s/jobs/282260153) about formatting. I know WDS updated it's [linting ruleset](https://github.com/WebDevStudios/WDS-Coding-Standards/tree/release-1.1) to include WordPress Docs. However, if I run PHPCBF it re-formats a lot of our PHP and looks terrible... almost completely unreadable by human standards. 

Yes - there are legit errors (no semi-colons), but the "spacing before closed" is a false-positive because we have `WPCS XSS OK.` in front of the closing `?>`.

Would love Travis to **WARN US** about formatting issues and **ERROR** on actual errors.

**Comment by gregrickaby** [comment]
After several more failures today, I was able to go through and eliminate all warnings.

Turns out there were a bunch of legit human errors. 

tl;dr: make sure your linting rulesets are up-to-date on your local machines and linting is turned on.
